src/lib.rs

```diff
@@ -1003,16 +1004,13 @@ trait OperationTrait {
 ////////////////////////
 
 /// Returns a string describing version information of the
-/// TensorFlow library. TensorFlow using semantic versioning.
+/// `TensorFlow` library. `TensorFlow` is using semantic versioning.
 pub fn version() -> std::result::Result<String, Utf8Error> {
     unsafe { CStr::from_ptr(tf::TF_Version()).to_str().map(|s| s.to_string()) }
 }
 
 ////////////////////////
 
-#[deprecated(note="Use Shape instead.")]
-type TensorShape = Shape;
```

Comment:
This was a bug and should have been public, which I fixed.  Please cancel this deletion.